### PR TITLE
github action to run upon pr

### PR DESCRIPTION
**Pull request by hchiam**
for #26 

demo: https://github.com/hchiam/test-blingblingjs/pull/1 😄

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Ava Tests on PR
+on: pull_request
```
Comment (hchiam) [inline .github/workflows/pr.yml]: i simply made a copy of `main.yml` and changed these 2 lines

**Review by hchiam — COMMENTED** [review]
ready to review

File: .github/workflows/pr.yml (hunk 1 above)
Comment (argyleink) [reply .github/workflows/pr.yml]: i think we may be able to reuse the first one and just add a comma and an array?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-multiple-events

**Comment by argyleink** [comment]
thanks for doing this! i think we can get away with even less code changed, see my comment about reusing `main.yml`! if that wont work, then what you've got here is great 👍🏻

File: .github/workflows/pr.yml (hunk 1 above)
Comment (hchiam) [reply .github/workflows/pr.yml]: yup, confirmed: https://github.com/hchiam/test-blingblingjs/pull/2

**Comment by hchiam** [comment]
yeah, one yaml file is cleaner - silly me overcomplicating things again haha

**Review by hchiam — COMMENTED** [review]
ready to review

**Review by argyleink — APPROVED** [review]
ty!

**Pull request merged**